### PR TITLE
Extra member info

### PR DESCRIPTION
**Pull request by ddabble**
*Depends on #355, and should change base to `dev` after that PR has been merged.*

Highlights:
* Added 6 extra member fields (f4f675017ad07b03152cefb91bf80d7ecab0f70b and 36a5f095b8d8ab05cb07d46f967514cf6a276927)
* Made [`Member.date_quit`](https://github.com/MAKENTNU/web/commit/78a6f238f36ec4242bd9a3396001b1bcb4f5f9b6#diff-844292acd3eab9b245dd5acf985f66769e14eb848d101d032b79ec02febcabbdR38) also apply to retirement (78a6f238f36ec4242bd9a3396001b1bcb4f5f9b6)
* Renamed `Member.email` -> `contact_email` (bfdfd4f5a9bcd2e6fb6c2b9891f6ec30906ec071)
* Added custom SemesterField (b97fc67e9265aa45a47edb95c52a9bf2a966a400)

### CODE REVIEW

**Review by zootos — APPROVED** [review]
Haven't ran and tested the code (due to some errors with installing the ldap package, that I can't be bothered fixing now), but have checked through all of the code. Some small suggestions-

File: internal/apps.py [hunk 1]
```diff
@@ -4,3 +4,7 @@
 class InternalConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'internal'
+
+    def ready(self):
+        # noinspection PyUnresolvedReferences
+        from . import signals
```
[the remark this answers is not in the document]
Comment (ddabble) [reply internal/apps.py:10]: Yes, good suggestion! 👌

File: internal/forms.py [hunk 2]
```diff
@@ -54,16 +54,24 @@ def save(self, commit=True):
...
+            'phone_number',
+            'study_program', 'ntnu_starting_semester', 'card_number',
+            'github_username', 'discord_username', 'minecraft_username',
+        ]
 
 
-class MemberQuitForm(forms.ModelForm):
+class MemberRetireForm(forms.ModelForm):
+    already_quit_or_retired_error_message = _(
+        "Member was not set as quit, as the member already has the status “quit” or “retired”."
```
[the remark this answers is not in the document]
Comment (ddabble) [reply internal/forms.py:67]: Huh, that's weird, thought I had checked it thoroughly for mistakes 😅🤔

**Comment by ddabble** [comment]
@zootos Thanks! 😄
Regarding the troubles installing `python-ldap`, just install a pre-built binary using `pip install <path to .whl file>` (see the instructions on [this page](https://www.python-ldap.org/en/python-ldap-3.3.0/installing.html#windows)) 🤠

**Pull request merged**